recipes/chromograph/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "chromograph" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
```

Comment:
Should `noarch: generic` be added in the build section?

Reply:
I downloaded the linuxartifact, and it fails to build when running `conda install -c ./packages chromograph`, I am hoping this would fix it, as the package will then be placed in a noarch folder within packages.

<img width="570" alt="image" src="https://github.com/bioconda/bioconda-recipes/assets/57712924/f25cfd86-b1fe-475e-9050-2a9799807d6f">

Reply:
noarch generic doesn't work because the package and dependencies require a compiler to install.

Reply:
Works for me
![Screenshot 2023-06-21 at 17 08 39](https://github.com/bioconda/bioconda-recipes/assets/20065894/3e1a028d-75ea-41e4-a5d7-a964476f15d8)

Reply:
Thank you for checking, good to see it works 👍